platform/android/detect.py

```diff
@@ -176,19 +185,35 @@ def configure(env: "SConsEnvironment"):
         )
     )
 
+    has_swappy = detect_swappy()
+    if not has_swappy:
+        print_warning(
+            "Swappy Frame Pacing not detected! It is strongly recommended you download it from https://github.com/darksylinc/godot-swappy/releases and extract it so that thirdparty/swappy-frame-pacing/arm64-v8a/libswappy_static.a can be found."
+        )
+        if not env["disable_swappy"]:
+            print_error(
+                "Build option `disable_swappy=yes` can be used to ignore missing Swappy dependency and build without it."
+            )
+            sys.exit(255)
+
     if get_min_sdk_version(env["ndk_platform"]) >= 24:
         env.Append(CPPDEFINES=[("_FILE_OFFSET_BITS", 64)])
 
     if env["arch"] == "x86_32":
         # The NDK adds this if targeting API < 24, so we can drop it when Godot targets it at least
         env.Append(CCFLAGS=["-mstackrealign"])
+        env.Append(LIBPATH=["../../thirdparty/swappy-frame-pacing/x86"])
```

Comment:
Should this also have the `has_swappy:` check?

Reply:
Yes. Fixed.